.github/workflows/release.yml

```
@@ -0,0 +1,112 @@
+      uses: actions/setup-go@v5
+      with:
+        go-version: '1.22.2'
+
+    - name: Run tests
+      run: go test -v ./...
+
+    - name: Build for multiple platforms
+      run: |
+        # Create dist directory
+        mkdir -p dist
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)